JFRWebSocket.h

```diff
@@ -109,6 +109,7 @@
  Set your own custom queue.
  Default setting is dispatch_get_main_queue.
  */
-@property(nonatomic, strong)dispatch_queue_t queue;
+@property(nonatomic)dispatch_queue_t queue;
```

Comment:
Could you rename this to something more explicit like "delegateQueue"? (see suggestion on callbackQueue as well). I found it to be confusing.